wordpress-mobile/WordPress-iOS
# Alert for deleting trashed posts

Fixes #6415 

To test:
For a post under Trashed, tap on delete.
Tap OK: The post should be deleted as usual
Tap Cancel: Alert is dismissed

Needs review: @aerych I am new to this, please let me know if I made a mistake!
